mscore/preferenceslistwidget.h

```diff
@@ -49,25 +51,32 @@ class PreferenceItem : public QTreeWidgetItem, public QObject {
 
       QString name() const {return _name;}
 
+   signals:
+      void editorValueModified();
       };
 
 //---------------------------------------------------------
 //   BoolPreferenceItem
 //---------------------------------------------------------
 class BoolPreferenceItem : public PreferenceItem {
-   private:
       bool _initialValue;
-      QCheckBox* _editor;
+      QCheckBox* _editor      {nullptr}; // make a QVariant out of these 3
+      QGroupBox* _editor2     {nullptr};
+      QRadioButton* _editor3  {nullptr};
```

Comment:
Can we have better names for those? Like `_editorCheckBox`, `_editorGroupBox`, `_editorRadioButton`?

Reply:
also, spaces around `nullptr`: `{ nullptr };`